Examples/test-suite/php/php_pragma_runme.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+require "tests.php";
+require "php_pragma.php";
+
+
+check::equal('1.5',(new ReflectionExtension('php_pragma'))->getVersion(),"1.5==version(php_pragma)");
+
+check::done();
+
+?>
```

Comment:
The red symbols github shows in the diff here mean there's no newline at the end of this file (if you hold the mouse pointer over them it'll even tell you).

It's a good idea to make sure text files have a newline at the end.  Some tools don't cope well if it's missing, and some editors automatically add one if there isn't one creating an unrelated whitespace change when someone using such an editor tries to update the file.

As a general rule of thumb, any differences in a source file you can't see are bad - so the last line should have a newline, there shouldn't be trailing whitespace at the end of any line, you shouldn't indent some lines with just spaces and use tabs in others (most code bases adopt a rule for how indents should be done), and there shouldn't be trailing blank lines at the end of the file.  There are cases where such whitespace can change the meaning of a file (e.g. `\` at the end of a line in a shell script or makefile is a line continuation, but if there's a space after it, it escapes that space instead!)